### DISCUSSION TITLE
How to send tab title change message using js

### DISCUSSION THREAD

**Discussion by zhch-hong** [microsoft/terminal#14683] [Q&A]
I want to set tab title dynamically in node.js script

**Comment by zadjii-msft** [comment]
Perhaps this doc might help: [Console Virtual Terminal Sequences | Window Title](https://learn.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences#window-title)

I'm afraid I won't be any help in how you set `ENABLE_VIRTUAL_TERMINAL_PROCESSING` from node, but once you do that, is should be as easy as something like:


```js
console.log("\x1b]2;My New Title\x1b\\")
```

**Reply by zhch-hong** [reply]
Sorry, I have absolutely no idea how to do this
